bitnami/nginx/Chart.yaml

```diff
@@ -25,4 +25,4 @@ name: nginx
 sources:
   - https://github.com/bitnami/bitnami-docker-nginx
   - https://www.nginx.org
-version: 13.0.0
+version: 13.0.1
```

Comment:
Could you increase the `minor` version instead? Since this PR is implementing a new feature.